projects/vdk-plugins/vdk-postgres/tests/conftest.py

```diff
@@ -19,17 +22,12 @@
 def _is_responsive(runner):
     try:
         result = runner.invoke(["postgres-query", "--query", "SELECT 1"])
-        if result.exit_code == 0:
-            return True
-    except:
-        return False
-
-
-@pytest.fixture(scope="session")
-def docker_compose_file(pytestconfig):
-    return os.path.join(
-        os.path.dirname(os.path.abspath(__file__)), "docker-compose.yml"
-    )
+    except Exception as e:
```

Comment:
Same applies here. I think its cleaner if we remove the try catch.

Reply:
Same answer as above. Let me see if I can re-write it to be clearer. Maybe add a coment